test/testcafe/framework/page-objects/PIVPageObject.js

```diff
@@ -18,14 +19,20 @@ export default class PIVPageObject extends BasePageObject {
   }
 
   getPageTitle() {
-    return this.form.getElement('.okta-form-title').textContent;
+    return this.getFormTitle();
   }
```

Comment:
We might be able to entirely remove this and replace all instances of it with `getFormTitle` directly, unless there's a lot of instances